.github/workflows/osrm-backend.yml

```diff
@@ -653,7 +653,7 @@ jobs:
   benchmarks:
     if: github.event_name == 'pull_request'
     needs: [format-taginfo-docs]
-    runs-on: ubuntu-24.04
```

Comment:
copy-paste from https://github.com/Project-OSRM/osrm-backend/pull/6975

Just to make benchmark results more stable, I'll remove it before merge.